spk/tvheadend/src/service-setup.sh

```diff
@@ -42,6 +47,18 @@ service_postinst ()
     $BIN/busybox --install $BIN
     $BIN/delgroup "${USER}" "users" >> ${INST_LOG}
     $BIN/deluser "${USER}" >> ${INST_LOG}
+
+    # EPG Grabber (zap2epg) - Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${PYTHONENV} >> ${INST_LOG}
+
+    # EPG Grabber (zap2epg) - Install the wheels/requirements
+    ${SYNOPKG_PKGDEST}/env/bin/pip install \
+             --no-deps --no-index --no-input --upgrade \
+             --force-reinstall --find-links \
+             ${WHEELHOUSE} ${WHEELHOUSE}/*.whl >> ${INST_LOG}  2>&1
+
+    # EPG Grabber (zap2epg) - Adjust permissions
+    set_unix_permissions "${PYTHONENV}"
```

Comment:
If package does not run without this, you must adjust permissions for DSM<7 only.

Reply:
There actually was no need for that ... removed.